community/concepts/deep-then-wide.md

```diff
@@ -0,0 +1,51 @@
+# deep then wide
```

Comment:
Same comment as on 'bikeshedding' entry, I think this should probably be capitalized.

Reply:
Done.